market/connector/docusign/demo/eSignatureDemo/src_hd/net/docusign/esignature/demo/DocUpload/DocUpload.xhtml

```diff
@@ -0,0 +1,37 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
+	xmlns:f="http://xmlns.jcp.org/jsf/core"
+	xmlns:h="http://xmlns.jcp.org/jsf/html"
+	xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
+	xmlns:ic="http://ivyteam.ch/jsf/component"
+	xmlns:p="http://primefaces.org/ui"
+	xmlns:pe="http://primefaces.org/ui/extensions">
+<h:body>
+	<ui:composition template="/layouts/basic-8.xhtml">
+		<ui:define name="title">DocUpload</ui:define>
+		<ui:define name="content">
+
+			<h3>Upload pdf document to be signed</h3>
```

Comment:
I do not see the reason why we focus so much on PDF documents throughout the whole show. 
In theory docusign can at ease handle a wide range of common formats such as DOCX, PDF, RTF.
They can even provide updated 'docx' or 'rft' files after signing.. with overlay pictures showing the signatures.

So I'd prefer to keep it open to these other formats. Or what was actually the driver to switch to PDF @ivy-bb ?

Reply:
I mean, that DocuSign returns the signed document always as pdf. If you upload an xx.rtf you get an xx.pdf back. Or are I wrong?

Reply:
I've  just checked again with a .doc. Worked perfectly! 
Don't now why I thought that only pdfs work...

Reply:
aha! you get the signed xxx.doc back from DocuSign but it is a pdf with a .doc extension. So it works that you can open it in the pdf viewer in an ivy Dialog but you can not open it with Word!

Reply:
mhh ... once I was able to actually also get a 'RTF' document with images embedded back. But I think for the showcase it could be anyway preferable to always download a PDF result file for archive purposes.

The error with the wrong final file extension can be adjust in the 'download doc' step. There we enforce a file extension as provided by the original file. I think we can just attach '.pdf' for the doc name there and we will be fine.
![docuSign_docDownload](https://user-images.githubusercontent.com/15085693/105689098-a2bc0c80-5efa-11eb-9433-cceb64404dd5.png)